docs-source/migrating-to-v4.md

```diff
@@ -0,0 +1,132 @@
+# Migration Guide
+
+## Migrating from v3 to v4
+
+Version 4 of both the Authentication API SDK as well as the Management API SDK are major breaking changes. This document will discuss the reason for these changes, as well as instructions on how to migrate to the new version.
+
+### OIDC Conformance
+
+The main reason for the breaking changes is related to the OIDC-conformance which was added to the Authentication API. Because of this, the behaviour of some of the existing Authentication API endpoints have changed, and other endpoints are being deprecated.
+
+For a full background and other details please refer to the official [OIDC-conformant authentication migration guide](https://auth0-docs-content-pr-3100.herokuapp.com/docs/api-auth/tutorials/migration).
```

Comment:
```suggestion
For a full background and other details please refer to the official [OIDC-conformant authentication migration guide](https://auth0.com/docs/api-auth/tutorials/adoption).
```

Where did that Heroku link come from? I'm wondering if that's part of another document out there.

Reply:
The v4 and v6 migration guides were already written I just split them into their own pages so not sure where it came from.